specify_pose/proposal.md

```diff
@@ -24,8 +24,8 @@ degrees) in radians, and authors may use different precisions in different
 circumstances, and, at a lower priority, (2) it is
 sometimes hard to visually separate translation from rotation.
 
-This proposal intends to resolve on point (1) by adding `//pose/rotation/@type`
-where degrees can be specified, and *could* address point (2) by structuring
+This proposal intends to resolve on point (1) by adding `//pose/@degrees`
+where degrees can be selected, and *could* address point (2) by structuring
 the element differently (see below).
```

Comment:
Should the intro have some sentences about quaternions?

Reply:
Yes, I think it should. I was hoping that folks who would like to see quaternions supported provide the motivations for doing so. The original proposal did not have a solid motivation for quaternions, so we decided to drop them, but we've had discussions to bring them back. It would be great if we can list out the motivations here, so we can justify the added complexity.

Reply:
There is some documented support in the following issues for adding quaternion support to URDF in the following issues:

* https://github.com/ros/urdfdom/issues/13
* https://github.com/ros/urdfdom/pull/123
* https://github.com/ros/urdfdom_headers/pull/51

In terms of justification, though I find Quaternions less visually inspectable, they are more straightforward to interpret than Euler angles (24 total flavors). This simplifies the portability of quaternions to other pieces of software and algorithms.

If using a camera calibration algorithm that yields Quaternion coefficients (such as [doi:10.1109/TIP.2011.2164421](https://doi.org/10.1109/TIP.2011.2164421)), it is most straightforward to directly specify those coefficients in the model file, rather than requiring a user to convert them to roll-pitch-yaw angles, with the risk of calculation errors and precision loss.

Reply:
shall I suggest text in this PR or open a new pull request with justification for quaternions?

Reply:
Either way works, but please feel free to make direct changes to this PR.